metrics/cpu_disabled.go

```diff
@@ -14,8 +14,8 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-//go:build ios || js || wasip1
-// +build ios js wasip1
+//go:build ios || js || wasip1 || tinygo.riscv
+// +build ios js wasip1 tinygo.riscv
```

Comment:
I think we can just disable it on TinyGo generally

Reply:
done